packages/mobile/src/verify/saga.ts

```diff
@@ -822,35 +839,64 @@ export function* getPhoneHashDetails() {
   }
 }
 
+function errorHandler(saga: any, sagaName: string) {
+  return function* (...args: any[]) {
+    try {
+      yield call(saga, ...args)
+    } catch (error) {
+      let errorString = error
+      let lastError = error
+      if (Array.isArray(error)) {
+        errorString = error.join(' - ')
+        lastError = error[-1]
+      }
+      Logger.error(TAG, `Error while running saga ${sagaName}`, error)
+      if (lastError?.toString() === FetchErrorTypes.QuotaExceededError) {
```

Comment:
This happens if the user runs out of Komenci quota. Can happen if they re-request attestations too many times.